src/elements/toast/toast.visual.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { html, nothing } from 'lit';
+
+import { describeEach, describeViewports, visualDiffDefault } from '../core/testing/private.js';
+
+import './toast.js';
+import '../button.js';
+import '../link.js';
+
+describe(`sbb-toast`, () => {
+  const cases = {
+    icon: [false, true],
+    action: ['dismissible', 'button', 'link'],
+  };
+
+  const positionCases = [
+    'top-start',
+    'top-center',
+    'top-end',
+    'bottom-start',
+    'bottom-center',
+    'bottom-end',
+  ];
+
+  describeViewports({ viewports: ['zero', 'medium'], viewportHeight: 300 }, () => {
+    describeEach(cases, ({ icon, action }) => {
+      it(
+        '',
```

Comment:
This leads to an underscore at the end in the test name. should we clean this somehow? (see link above)

Reply:
I added a `trim()` before the title transformation. It solves the problem and should not lead to any regression (except for the tests that ended with a white space)